src/main/java/htsjdk/samtools/cram/compression/ExternalCompression.java

```diff
@@ -1,6 +1,7 @@
-package htsjdk.samtools.cram.io;
```

Comment:
I moved compression classes out of the `encoding` package.  This move felt appropriate as well.